src/widget.py

```diff
@@ -4,16 +4,22 @@
 
 
 def mask_account_card(numbers: Any) -> Any:
```

Comment:
Не верно расставлены тайпхинты
```suggestion
def mask_account_card(numbers: str) -> str:
```